### PR TITLE
Hide Gary by default, add an endpoint for unhiding service users when needed

### PR DESCRIPTION
**Pull request by benilovj**
To unhide a hidden service user, access:

`/show-case/<CRN>`

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -16,4 +16,13 @@ router.get('/switch-provider/:newProvider', function (req, res) {
   res.redirect('/cases')
 })
 
+router.get('/unhide/:CRN', function (req, res) {
```
Comment (fofr) [inline app/routes.js:19]:
```suggestion
router.get('/show-case/:CRN', function (req, res) {
```
?

**Comment by fofr** [comment]
An alternative approach might be to have the data stored somewhere other than the default session data, then to pull into the session at the point where it's enabled.

**Review by fofr — APPROVED** [review]
1 comment about "unhide" vs "show", but 👍

File: app/routes.js (hunk 1 above)
Comment (benilovj) [reply app/routes.js]: Agreed, thanks.

**Comment by benilovj** [comment]
> An alternative approach might be to have the data stored somewhere other than the default session data, then to pull into the session at the point where it's enabled.

Yep. The approach I've taken is more verbose but has the benefit of being very explicit and keeping the data working as before...

**Pull request merged**